src/containers/WebComponentLoader.jsx

```diff
@@ -31,24 +32,25 @@ const WebComponentLoader = (props) => {
   const {
     assetsIdentifier,
     authKey,
-    identifier,
     code,
-    senseHatAlwaysEnabled = false,
-    instructions,
-    withProjectbar = false,
-    projectNameEditable = false,
-    withSidebar = false,
-    sidebarOptions = [],
-    theme,
-    outputPanels = ["text", "visual"],
     embedded = false,
     hostStyles, // Pass in styles from the host
-    showSavePrompt = false,
+    identifier,
+    instructions,
     loadRemixDisabled = false,
-    readOnly = false,
     outputOnly = false,
+    outputPanels = ["text", "visual"],
     outputSplitView = false,
+    projectNameEditable = false,
+    reactAppApiEndpoint = process.env.REACT_APP_API_ENDPOINT,
```

Comment:
💪 nice!! So given this is a default fallback do you reckon its worth keeping this env var in the `.env.example` and CI/CD so it'll work as is but can be overridden in the `editor-wc` props?

Reply:
Sure! done in [727da9c](https://github.com/RaspberryPiFoundation/editor-ui/pull/1124/commits/727da9c16abb1638406c76afbf58544dab6ddbe8) :)